### ISSUE TITLE
Hitomi downloader truncate filenames after 100 characters.

### ISSUE THREAD

**Issue by doppler56** [KurtBestor/Hitomi-Downloader#6205]
Both Windows and Linux both have a limit of 256 characters.  Truncating after 200 would solve many pathname problems I have with downloaded files.  100 character limit is way too short.

https://it.cornell.edu/shared-file/windows-file-name-or-destination-path-you-specified-not-valid-or-too-long

**Comment by doppler56** [comment]
Maybe the filename size can be user adjustable.  Through preferences.  Does it have to be a hard number ?

**Comment by KurtBestor** [comment]
Filenames have a 256-character limit, not 100.

**Comment by KurtBestor** [comment]
No, Hitomi downloader has 256-character limit

**Comment by doppler56** [comment]
Then there is a bug.  Hitomi Downloader (HD) truncates after 100.  You can easily find a long filename on hitomi.la website.
I have been using HD for going on 2 years now.  Always truncated.

**Comment by KurtBestor** [comment]
You mean folder name?

**Comment by doppler56** [comment]
yes, folders are always the longest.  Rarely see filename longer than 20 characters.  Sorry for the confusion filename and dirnames I see them the same.  It's just a name.  Foldername and dirname is the same too, only the O/S describes them differently.  Note: The easy confusion.
